wpilibc/src/main/native/include/frc/AprilTagFieldLayout.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <vector>
+
+#include <wpi/SymbolExports.h>
+
+#include "frc/DriverStation.h"
+#include "frc/apriltag/AprilTagUtil.h"
+#include "frc/geometry/Pose3d.h"
+
+namespace wpi {
+class json;
+}  // namespace wpi
+
+namespace frc {
+class WPILIB_DLLEXPORT AprilTagFieldLayout {
+ public:
+  AprilTagFieldLayout() = default;
+
+  explicit AprilTagFieldLayout(
+      const std::vector<AprilTagUtil::AprilTag>& apriltags);
+
+  const std::vector<AprilTagUtil::AprilTag>& GetTags() const;
```

Comment:
Is this getter needed?

Reply:
Not anymore.

Reply:
I was wrong, it's still used in the test, however I'm unsure if the equal operator should just be implemented for it instead.

Reply:
There should be one implicitly defined already.

Reply:
Removing the `.getTags` from the tests results in a error, it will need to be defined explicitly I believe

Reply:
Then you should be able to do so with `bool operator=(const AprilTagFieldLayout&) = default;`.

Reply:
With a recent refactor, this function is no longer used.